media/js/firefox/enterprise-sla.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+window.Mzp.Details.init('.mzp-c-article .mzp-c-details > h3');
+
+(function() {
+    'use strict';
+
+    var sidebarLinks = document.querySelectorAll('#sidebar-menu a[href^="#"]');
+
+    for (var i = 0; i < sidebarLinks.length; i++) {
+        sidebarLinks[i].addEventListener('click', function() {
+            // Extract the target element's ID from the link's href.
+            var target = this.getAttribute('href').replace( /.*?(#.*)/g, '$1' );
+
+            if (document.querySelector(target + ' > .is-summary > button[aria-expanded=false]')) {
```

Comment:
Suggest referencing this DOM query via a variable for resue on the line below, rathern than querying twice.